packages/imperative/src/doc/IImperativeConfig.ts

```diff
@@ -286,6 +286,28 @@ export interface IImperativeConfig {
      */
     pluginSummary?: string;
 
+    /**
+     * If the project you are configuring is an Imperative plugin,
+     * you can assign the first steps to the plugin, where it will be
+     * visible when running the firststeps command in the root plugins
+     * level help for the core CLI.
+     *
+     * The first steps are a short list of steps required to set up
+     * your plugin for use.
+     *
+     * Note: For plugins only. Ignored when specified on a core/base CLI
+     *
+     * @example
+     * const config: IImperativeConfig = {
+     * ...
+     * pluginFirstSteps: "a list of first steps for your plugin"
+     * ...
+     * }
+     * @type {string}
+     * @memberof IImperativeConfig
+     */
+    pluginFirstSteps?: string;
```

Comment:
Text input here seems ok.
I'm curious if we would eventually support a custom "handler" (similar to `pluginHealthCheck`) that gives more flexibility to plugin developers on what they can do for the users in terms of setup/configuration.